README.md

```diff
@@ -1,199 +1,243 @@
-
-```php
-<?php
-
-namespace App\Services\FeaturePolicy\Policies;
-
-use Mazedlx\FeaturePolicy\Directive;
-use Mazedlx\FeaturePolicy\Policies\Basic;
-
-class MyFeaturePolicy extends Basic
-{
-    public function configure()
-    {
-        parent::configure();
-
-        $this->addDirective(Directive::GEOLOCATION, 'www.awesomesite.com')
-            ->addDirective(Directive::FULLSCREEN, 'www.awesomesite.com');
-    }
-}
-```
-
-Don't forget to change the `policy` key in the `feature-policy` config file to the class name fo your policy (e.g. `App\Services\Policies\MyFeaturePolicy`).
-
-## Testing
-
-You can run all tests with:
-
-```bash
-$ composer test
-```
-
-## Changelog
-
-Please see [CHANGELOG](https://github.com/mazedlx/laravel-feature-policy/blob/master/CHANGELOG.md) for more information what has changed recently.
-
-## Contributing
-
-Please see [CONTRIBUTING](https://github.com/mazedlx/laravel-feature-policy/blob/master/CONTRIBUTING.md) for details.
-
-### Security
-
-If you discover any security related issues please email mazedlx@gmail.com instead of using the issue tracker.
-
-## Credits
-
-- [Freek van der Herten](https://github.com/freekmurze)
-- [Thomas Verhelst](https://github.com/TVke)
-- [All Contributors](https://github.com/mazedlx/laravel-feature-policy/contributors)
-
-## Support
-
-If you like this package please feel free to star it.
-
-## License
-
-The MIT License (MIT). Please see [LICENSE](https://github.com/mazedlx/laravel-feature-policy/blob/master/LICENSE.md) for more information.
+# Configure the browsers abilities
+
+[![Latest Version on Packagist](https://img.shields.io/packagist/v/mazedlx/laravel-feature-policy.svg?style=flat-square)](https://packagist.org/packages/mazedlx/laravel-feature.policy)
+[![Tests](https://github.com/mazedlx/laravel-feature-policy/actions/workflows/test.yml/badge.svg)](https://github.com/mazedlx/laravel-feature-policy/actions/workflows/test.yml)
+[![Analyse and format](https://github.com/mazedlx/laravel-feature-policy/actions/workflows/code-quality.yml/badge.svg)](https://github.com/mazedlx/laravel-feature-policy/actions/workflows/code-quality.yml)
+[![Total Downloads](https://img.shields.io/packagist/dt/mazedlx/laravel-feature-policy.svg?style=flat-square)](https://packagist.org/packages/mazedlx/laravel-feature-policy)
+
+
+The Permissions-Policy, which [previously](https://docs.w3cub.com/http/headers/feature-policy) was known as the Feature-Policy.
+But since it came out of draft, it was renamed to "Permissions-Policy".  
+The "Permissions-Policy" is an HTTP header which can be used to restrict the abilities of a browser.
+
+Where the Content-Security-Policy focuses on security, the "Permissions-Policy" focuses on allowing or disabling the abilities of the browser.  
+This can be done though the HTTP header, which this package focuses on, but it can also do this through the `allows` attribute on the `iframe` element.
+
+<details>
```

Comment:
This creates a collapsable section. I've used this throughout the readme, to hide large parts. That, while important, don't make the overall reading easy. So its available if someone wants to see it, otherwise it stays hidden.